tests/rest/reports_info.go

```diff
@@ -0,0 +1,170 @@
+/*
+Copyright © 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tests
+
+import (
+	httputils "github.com/RedHatInsights/insights-operator-utils/http"
+	"github.com/RedHatInsights/insights-results-aggregator-data/testdata"
+	"github.com/verdverm/frisby"
+
+	"github.com/RedHatInsights/insights-results-aggregator/server"
+	"github.com/RedHatInsights/insights-results-aggregator/types"
+)
+
+// constructURLForReportInfoForOrgCluster function constructs an URL to access
+// the endpoint to retrieve results metadata for given cluster from selected
+// organization
+func constructURLForReportInfoForOrgCluster(organizationID string,
+	clusterID string, userID types.UserID) string {
+	return httputils.MakeURLToEndpoint(apiURL,
+		server.ReportMetainfoEndpoint, organizationID, clusterID, userID)
+}
+
+// checkReportInfoEndpointForKnownOrganizationAndKnownCluster check if the
+// endpoint to return report metadata works as expected
+func checkReportInfoEndpointForKnownOrganizationAndKnownCluster() {
```

Comment:
I would create a function like:
```go
setStatus(f *Frisby, status int) {
	f.Send()
	f.ExpectStatus(status)
	f.ExpectHeader(contentTypeHeader, ContentTypeJSON)
	f.PrintReport()
}
```
to avoid the repetition in all these functions.

Reply:
Or even better:
```go
setStatus(msg, url string, status int) {
        f := frisby.Create(msg).Get(url)
	f.Send()
	f.ExpectStatus(status)
	f.ExpectHeader(contentTypeHeader, ContentTypeJSON)
	f.PrintReport()
}
```

Reply:
PTAL at this proposal: https://github.com/tisnik/insights-results-aggregator/pull/1